### ISSUE TITLE
Juliaup lies about versions

### ISSUE THREAD

**Issue by tecosaur** [JuliaLang/juliaup#703]
I've been pressured to actually submit this bug report instead of just laughing about it (cc: @LilithHafner)

#### The issue in brief

```
~$ julia +1.10
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.9.2 (2023-07-05)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |
```

In stacktraces we see that the Julia path is `juliaup/julia-1.9.2+0.x64.linux.gnu/`.

I'm not sure what extra info is wanted, but if I do `juliaup status` I see this:

```
~$ juliaup status
 Default  Channel  Version                        Update 
---------------------------------------------------------
          1.10     1.10.0-alpha1+0.x64.linux.gnu         
          1.8      1.8.5+0.x64.linux.gnu                 
          1.9      1.9.2+0.x64.linux.gnu                 
       *  release  1.9.2+0.x64.linux.gnu      
```

**Comment by davidanthoff** [comment]
I can't reproduce this... It seems that the launcher is ignoring the command line override, i.e. `julia +1.10` on your system seems to ignore the `+1.10`. Can you run `which julia` and report the output here?

**Comment by tecosaur** [comment]
`which julia` gives `/usr/bin/julia`, which is a binary installed by the `juliaup` package (https://build.opensuse.org/package/show/openSUSE:Factory/juliaup).

`juliaup --version` gives `1.11.0`.

**Comment by davidanthoff** [comment]
Hm, so that is weird: version 1.11.0 was never released, it was an intermediate test build. I don't know who is doing the openSUSE packaging, but they should not push out versions that are marked as prerelease here on GitHub.

Plus, they seem to be behind the latest released version, which is 1.11.15.

For now I'm going to close this, most likely the problem here is that someone is packaging and deploying incorrect Juliaup versions on openSUSE.
